samples/Exporters/Console/TestRedis.cs

```diff
@@ -39,13 +49,7 @@ internal static object Run(string zipkinUri)
                         o.ServiceName = "redis-test";
                         o.Endpoint = new Uri(zipkinUri);
                     })
-                    // TODO: Uncomment when we change Redis to Activity mode
-                    // .AddInstrumentation(t =>
-                    // {
-                    //    var instrumentation = new StackExchangeRedisCallsInstrumentation(t);
-                    //    connection.RegisterProfiler(instrumentation.GetProfilerSessionsFactory());
-                    //    return instrumentation;
-                    // })
+                    .AddRedisInstrumentation(connection)
```

Comment:
Perhaps a good idea to use `MaxFlushInterval` in this example so people can easily find it exists :)

Reply:
just updated!